src/karmen_frontend/src/components/listings/printers-table.js

```diff
@@ -155,36 +167,55 @@ const PrintersTableRow = ({
   );
 };
 
-const PrintersTable = ({
-  orguuid,
-  onPrinterUpdate,
-  onPrinterDelete,
-  loadPrinters,
-  printersLoaded,
-  printersList,
-}) => {
-  return (
-    <NoPaginationListing
-      defaultOrderBy="+name"
-      loadItems={() => loadPrinters(["job", "status", "webcam", "lights"])}
-      itemsLoaded={printersLoaded}
-      items={printersList}
-      enableFiltering={true}
-      sortByColumns={["name"]}
-      filterByColumns={["name"]}
-      rowFactory={(p) => {
-        return (
-          <PrintersTableRow
-            key={p.uuid}
-            orguuid={orguuid}
-            printer={p}
-            onPrinterUpdate={onPrinterUpdate}
-            onPrinterDelete={onPrinterDelete}
-          />
-        );
-      }}
-    />
-  );
-};
+class PrintersTable extends React.Component {
+  state = { canUpdate: true };
+
+  shouldComponentUpdate(nextProps, nextState, nextContext) {
+    return this.state.canUpdate;
+  }
+  constructor(props) {
+    super(props);
+    this.canUpdateTable = this.canUpdateTable.bind(this);
```

Comment:
"Can update table" feels like a question not while the method really does an action.

Could you rename it to `pauseUpdates` or something like that? It can be that short because it is already in the context of printersTable?